mibs/hp/HP-ICF-POE

```diff
@@ -0,0 +1,1455 @@
+-- Copyright (C) 2004-2005, 2007, 2009-2010, 2012-2013, 2015-2017

```

Comment:
can you rename this file `HP-ICF-POE-MIB` ?

Reply:
Done, should be matching the MIB definition name now